docker-compose.yml

```diff
@@ -82,6 +82,8 @@ services:
         condition: service_completed_successfully
     profiles:
       - unit-tests
+    volumes:
+      - ./:/vuln4shift
```

Comment:
@yungbender will there be needed any changes in building the test image if we just mount the git dir (like removing some unnecessary steps)? Or is it ok?